scripts/run-android.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+set -m # needed to access jobs
+
+GIT_ROOT=$(cd "${BASH_SOURCE%/*}" && git rev-parse --show-toplevel)
+
+# generates android debug build when we pass the debug flag
+"${GIT_ROOT}"/scripts/build-android.sh
```

Comment:
Is here ever a use case for not passing debug build and env type?
```suggestion
export BUILD_ENV=debug
export BUILD_TYPE=debug
"${GIT_ROOT}"/scripts/build-android.sh
```
It only makes sense to set those in the `Makefile` if there is a use case for overriding them.

Reply:
addressed with : https://github.com/status-im/status-mobile/pull/18784/commits/8a330cc8708d68b58e1ab943eb1f06dfe0c4e46d